scripts/rename-r2-to-4digit.sh

```
@@ -0,0 +1,53 @@
+echo "バケット: $BUCKET_NAME"
+echo "公開URL: $R2_PUBLIC_URL"
+echo ""
+
+# 方法1: wrangler経由でコピー＆削除
+# wrangler r2 object get/put/delete を使用
+
+# まず、3桁ファイルが存在するかHTTPで確認
+echo "=== 3桁ファイルの存在確認 ==="
+for i in $(seq -w 1 200); do
+  # 3桁形式 (001, 002, ..., 200)
+  three_digit=$(printf "%03d" $i)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)